e2e/tests/deploy/deploy.go

```diff
@@ -194,4 +194,77 @@ var _ = DevSpaceDescribe("deploy", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(out, "test")
 	})
+
+	ginkgo.It("should deploy helm applications concurrently", func() {
```

Comment:
IMHO test case name should be renamed to "should deploy applications concurrently". Because this feature supposes to work with both kubectl and helm deployment configs.